R/derive_param_computed.R

```diff
@@ -117,13 +117,18 @@
 #' @param keep_nas Keep observations with `NA`s
 #'
 #'   If the argument is set to `TRUE`, observations are added even if some of
-#'   the values contributing to the computed value are `NA`.
+#'   the values contributing to the computed value are `NA` (see Example 1b).
+#'
+#'   If the argument is set to a list of variables, observations are added even
```

Comment:
So keep_nas can be TRUE, FALSE or a list of variables?  I like not adding another argument but do we do this in other functions - is this common in other packages?

Reply:
We have something similar for the `parameters` argument in `derive_param_computed()`. I'm not sure if we do this in other functions.

I don't know how common this is in other packages but there are definitely examples in other packages. Consider for example `dplyr::select()`. It accepts both symbols and strings.

Reply:
Cool Cool!  That is good example.

All good for me then.